### PR TITLE
What's new 11.7.1.0

### PR DESCRIPTION
**Pull request by krzychu124**
New
- Dead-end lane connections #1613, #1213 
- Reset lane arrows when the user delete lane connections from selected node #1613

Fixed
- Cargo Trucks ignoring vehicle restrictions #1661 
- Parking AI - _"thinking about a parking spot"_ near pedestrian streets #1661, #1655 
- Parking AI confused in large parking lots #1626, #1394, #924, #423, #352
- Internal issues causing desync when other mod patched `Citizen` / `CitizenUnit` size in `CitizenManager` #1599

Updated
- Slider value moved to label instead of tooltip #1660 
- Hot-reload save - debug improvements #1635 
- Show DLC icon near the option if the feature requires DLC mechanics to work (previously the option was invisible without DLC) #1657 

TO DO
- [ ] Translations update

### CODE REVIEW

File: TLM/TLM/Util/VersionUtil.cs [hunk 1]
```diff
@@ -35,10 +35,10 @@ public static class VersionUtil {
         // we could alternatively use BuildConfig.APPLICATION_VERSION because const values are evaluated at compile time.
         // but I have decided not to do this because I don't want this to happen automatically with a rebuild if
         // CS updates. these values should be changed manaually so as to force us to acknowledge that they have changed.
-        public const uint EXPECTED_GAME_VERSION_U = 197256464U;
+        public const uint EXPECTED_GAME_VERSION_U = 197256976U;
 
         // see comments for EXPECTED_GAME_VERSION_U.
-        public static Version ExpectedGameVersion => new Version(1, 15, 0, 5);
```
Comment (kianzarrin) [inline TLM/TLM/Util/VersionUtil.cs:41]: was there another game update or this is just from beta to release?

**Comment by kianzarrin** [comment]
what about clearing lane connections from settings
also removing lane connections resets lane arrows (bonus in my dead end PR).

File: TLM/TLM/Util/VersionUtil.cs (hunk 1 above)
Comment (krzychu124) [reply TLM/TLM/Util/VersionUtil.cs:41]: Current version is 1.15.0-f7. I think I forgot to bump it when releasing `11.7.0.1` fix. That update didn't change anything for mods, just a tiny bugfix in the game guide popups not showing.

**Comment by krzychu124** [comment]
> what about clearing lane connections from settings also removing lane connections resets lane arrows (bonus in my dead end PR).

Clearing connections feature has been released in `11.7.0.1` (see left side of diff)
I forgot about that bonus thingy 👍

**Review by kianzarrin — APPROVED** [review]

**Review by kvakvs — APPROVED** [review]

**Pull request merged**